### ISSUE TITLE
Hints hidden by please-let-us-record-and-sell-your-data modal dialog

### ISSUE THREAD

**Issue by glacambre** [tridactyl/tridactyl#2450]
Tridactyl version: 1.19.0pre3895-1b63ea2
Firefox version: Mozilla Firefox 76.0.1
Operating system: linux

Steps to reproduce:
1) Disable any adblocker/js-blocker you may have
2) Go to https://www.theatlantic.com/ideas/archive/2020/06/american-cities-are-not-battlespaces/612553/
3) Attempt to kill the modal with `;k`. everything turns green/yellow but hints are invisible.

**Comment by ozankececioglu** [comment]
I am new to tridactyl and love it already, except hinting those dialogs. Tridactyl seems to be able to highlight over dialog elements, but the labels stuck behind the dialog. Many cookie popups are affected in my experience. Github left bar menu is also included, see pic below.


![image_171](https://github.com/user-attachments/assets/6c2afcc3-7d01-4a3c-8fea-bff60781d793)

**Comment by bovine3dom** [comment]
I'm wondering if we can fix this by editing any CSS that has z-index: (max int) to subtract one when we enter hint mode

**Comment by bovine3dom** [comment]
Yeah, `dialog` is a new feature and there's not much we can do about it easily. Most sites will be using the z-index still.

I'm wondering if we can somehow trick websites into thinking that `dialog` isn't available so they fall back to an old method.

**Comment by bovine3dom** [comment]
Fixed in #5438
